Packs/Akamai_SIEM/Integrations/Akamai_SIEM/README.md

```diff
@@ -43,6 +43,9 @@ A WAF (web application firewall) is a filter that protects against HTTP applicat
     | Fetch limit | False | Limit on the number of incidents retrieved in a single fetch. The maximum is 80k.|
     | Akamai Page size | False | The number of events to fetch per request to akamai (multiple requests are made for each fetch). If you're getting aggregated delays, increase the number. The maximum is 80,000. |
     | Skip events decoding | False | Use this parameter to avoid decoding the http message and attack data fields and speed up the ingestion rate. |
+    | Long running instance | False | This is a beta feature for testing new mechanism for fetch events. Use this param only if advised by CS. Make sure this feature is not used with fetch events configured in the integration params and that there's no config ID used for 2 different instances / features.  |
```

Comment:
add a mark here that the point here is high performance, we didnt signify that. We should also add a note on the fetch events command that if their instance is expected to have high rate of data that they should check this

Reply:
But @anara123 said this should be something that should be advised by cs.

Reply:
we should still explain what this does

Reply:
I did, I just didn't add the "check the long running option if you have a lot of events" to the fetch events additional info